.circleci/config.yml

```diff
@@ -83,46 +83,17 @@ jobs:
             ./codecov
       - store_test_results:
           path: ./
-  sign_release:
-    docker:
-      - image: cimg/base:2020.01
-    steps:
-      - attach_workspace:
-          at: ~/project
-      - run:
-          name: "Ensure GnuPG is available"
-          command: gpg --version
-      - run:
```

Comment:
This step should be kept in the build process

Reply:
This is still incorrect, you should remove the `sign_release` **job** as it is done by goreleaser now. But you have to move the **step** `Import cloudskiff signing key` to the release **job** 

```yaml
  release:
    steps:
      - run:
          name: "Import cloudskiff signing key"
          command: |
            echo ${SIGNINGKEY} | base64 -d > signingkey
            gpg --import signingkey
      - run:
          name: "Build and publish release"
          command: make release
```

The key is stored in env as base64 string and thus need to be decoded and imported in gpg